redaxo/src/addons/backup/pages/export.php

```diff
@@ -154,7 +154,7 @@
 $tables = rex_sql::factory()->getTables();
 foreach ($tables as $table) {
     $tableSelect->addOption($table, $table);
-    if ($table === rex::getTable('user') || $table === rex::getTable('user_session')) {
+    if (in_array($table, [rex::getTable('user'), rex::getTable('user_passkey'), rex::getTable('user_session')], true)) {
```

Comment:
hat das einen impact auf import/export (exporte aus vorversionen hier nicht importierbar, oder exporte aus 5.15 nicht in 5.14 importierbar o.ä.)?

Reply:
Importe zwischen den Versionen sind allgemein problematisch, und sollte man aktuell nicht tun. Also aktuell sollte man nur Backups in der gleichen Version wieder einspielen. Das Problem schiebe ich schon lange vor mir her, haben dazu auch Issues glaube ich.